sfdx/package.json

```diff
@@ -1,42 +1,63 @@
 {
-				"name": "sfdx-mobilesdk-plugin",
-				"version": "7.1.0",
-				"description": "Sfdx plugin for creating mobile apps based on the Salesforce Mobile SDK",
-				"main": "index.js",
-				"keywords": [
-								"mobilesdk",
-								"salesforce",
-								"mobile",
-								"sdk",
-								"salesforcedx",
-								"salesforce-dx"
-				],
-				"homepage": "https://github.com/forcedotcom/SalesforceMobileSDK-Package",
-				"bugs": "https://github.com/forcedotcom/SalesforceMobileSDK-Package/issues",
-				"licenses": [
-								{
-												"type": "Salesforce.com Mobile SDK License",
-												"url": "https://github.com/forcedotcom/SalesforceMobileSDK-iOS/blob/master/LICENSE.md"
-								}
-				],
-				"engines": {
-								"node": ">=7.0.0"
-				},
-				"dependencies": {
-								"shelljs": "~0.7.0"
-				},
-				"repository": {
-								"type": "git",
-								"url": "https://github.com/forcedotcom/SalesforceMobileSDK-Package.git"
-				},
-				"author": {
-								"name": "Wolfgang Mathurin",
-								"email": "wmathurin@salesforce.com"
-				},
-				"contributors": [
-								{
-												"name": "Bharath Hariharan",
-												"email": "bhariharan@salesforce.com"
-								}
-				]
+    "name": "sfdx-mobilesdk-plugin",
+    "version": "7.1.0",
+    "description": "Sfdx plugin for creating mobile apps based on the Salesforce Mobile SDK",
+    "keywords": [
+        "mobilesdk",
+        "salesforce",
+        "mobile",
+        "sdk",
+        "salesforcedx",
+        "salesforce-dx",
+        "oclif-plugin",
+        "sfdx-plugin"
+    ],
+    "homepage": "https://github.com/forcedotcom/SalesforceMobileSDK-Package",
+    "bugs": "https://github.com/forcedotcom/SalesforceMobileSDK-Package/issues",
+    "licenses": [
+        {
+            "type": "Salesforce.com Mobile SDK License",
+            "url": "https://github.com/forcedotcom/SalesforceMobileSDK-iOS/blob/master/LICENSE.md"
+        }
+    ],
+    "engines": {
+        "node": ">=8.10.0"
+    },
+    "dependencies": {
+        "@oclif/command": "1",
+        "@oclif/config": "1",
+        "@oclif/errors": "1",
+        "@oclif/plugin-help": "2",
+        "@salesforce/core": "^1.1.2",
+        "globby": "^9.0.0",
+        "shelljs": "0.7.0"
+    },
+    "devDependencies": {
+        "@oclif/dev-cli": "^1.21.2"
+    },
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/forcedotcom/SalesforceMobileSDK-Package.git"
+    },
+    "author": {
+        "name": "Wolfgang Mathurin",
+        "email": "wmathurin@salesforce.com"
+    },
+    "contributors": [
+        {
+            "name": "Bharath Hariharan",
+            "email": "bhariharan@salesforce.com"
+        }
+    ],
+    "files": [
+        "/shared/**/*.js",
+        "/oclif/**/*.js",
+        "/bin"
+    ],
+    "oclif": {
+        "commands": "./oclif"
+    },
+    "scripts": {
+        "prepack": "node_modules/.bin/oclif-dev manifest"
```

Comment:
It's key to run oclif-dev in prepack. I had to modify pack.js and test_force to get the generated manifest in the correct places.